File: 0520-detect-capital/0520-detect-capital.js
/**
 * @param {string} word
 * @return {boolean}
 */
var detectCapitalUse = function(word) {
    let count = 0;
    const n = word.length;
    for (let i = 0; i < n; i++) {
        if (word.charCodeAt(i) <= 90) {
            count++;
        }
    }
    return count === n || count === 0 || count === 1 && word.charCodeAt(0) <= 90;
};

File: 0520-detect-capital/0520-detect-capital.cpp
class Solution {
public:
    bool detectCapitalUse(string word) {
        int count = 0;
        int n = word.size();
        for (int i = 0; i < n; i++) {
            if (int(word[i]) <= 90) {
                count++;
            }
        }
        return count == n || count == 0 || count == 1 && int(word[0]) <= 90;
    }
};